architecture/routing.adoc

```diff
@@ -10,17 +10,38 @@
 toc::[]
 
 == Overview
-Routers provide external DNS mapping and load balancing to services over protocols that pass distinguishing information directly to the router (HTTP, HTTPS, and TLS with SNI). With a service, the router uses the service selector to watch the endpoints that support the service, bypassing its logic and replacing it with the router's own. Routers subscribe to a configuration and automatically update themselves with any changes, and may be containerized or virtual (converting those changes to API calls to a system like an F5).
+Routers provide external DNS mapping and load balancing to services over the
+following protocols that pass distinguishing information directly to the router:
 
-Other capabilities exist to load-balance a service within a cluster. These services are exposed via a configurable link relation between different services, and ensure a set of services can be available. Deployments may choose to use these services as local proxies for each host, or to reuse the shared routing infrastructure.
+- HTTP
+- HTTPS
+- WebSockets
+- TLS with SNI
 
-While configuring a router is up to the OpenShift administrator, setting the route types of each individual application is up to the application developer.
+NOTE: WebSocket traffic uses the same route conventions and supports the same
+TLS termination types as other traffic.
 
-== Route Types
-Routes can be either secure or unsecure. Secure routes provide the ability to use different types of TLS termination to serve certificates to the client. To create a secure route, specify the TLS termination of the route in a json file.
+With a link:kubernetes_model.html#service[service], the router uses the service
+selector to watch the endpoints that support the service, bypassing its logic
```

Comment:
Removed "With a service", and moved the link to `kubernetes_model.html#service` up to the first sentence.